internal/format/util.go

```diff
@@ -75,7 +75,7 @@ func getCloseTokenForOpenToken(kind ast.Kind) ast.Kind {
 	return ast.KindUnknown
 }
 
-func getLineStartPositionForPosition(position int, sourceFile *ast.SourceFile) int {
+func GetLineStartPositionForPosition(position int, sourceFile *ast.SourceFile) int {
 	lineStarts := scanner.GetLineStarts(sourceFile)
```

Comment:
Not for this PR probably but its very suspect how this asks for the line starts again, and it's probably not even the right line starts given this is LS code. We need to audit all of this again but doesn't have to be this PR obviously